app/views/api/v2/errors/unsupported_media_type.json.rabl

```diff
@@ -0,0 +1,3 @@
+object false
+
+node(:message) { _("Media type in 'Content-Type: %s' is unsupported in API v2 for POST and PUT requests. Please use 'Content-Type: application/json'.") % request.content_type }
```

Comment:
should this be a media type at the end of the line too?

Reply:
No, it should not, the header should be still printed out whole in the error message I believe (`content_type` should include )
Or we can rephrase this to `Media type '%s' is unsupported`